Support → Report Issue msg fails more often than it works

### Repro
Settings → Support → Report Issue → enter email and message → Send

### Remarks
Seems to be an API timeout issue, from the logs:
```js
APP              to.bitkit       E  2026-01-15 16:13:51.457 ERROR   [LogsRepo.kt:53]                     Connect timeout has expired [url=https://synonym.to/api/chatwoot, connect_timeout=30000 ms] [ConnectTimeoutException='Connect timeout has expired [url=https://synonym.to/api/chatwoot, connect_timeout=30000 ms]'] - SupportRepo (Explain with AI)

                                    Caused by: java.net.SocketTimeoutException: failed to connect to synonym.to/34.118.30.82 (port 443) from /192.168.68.65 (port 46422) after 30000ms
```

--- a/app/src/main/java/to/bitkit/di/HttpModule.kt
+++ b/app/src/main/java/to/bitkit/di/HttpModule.kt
@@ -46,8 +46,8 @@ object HttpModule {
     @Suppress("MagicNumber")
     private fun HttpTimeoutConfig.defaultTimeoutConfig() {
         requestTimeoutMillis = 60_000
-        connectTimeoutMillis = 30_000
-        socketTimeoutMillis = 30_000
+        connectTimeoutMillis = 60_000
+        socketTimeoutMillis = 60_000
     }
 
     private fun LoggingConfig.defaultLoggingConfig() {

